SmallStatusWidget/GlucoseView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  GlucoseView.swift
+//  Loop
+//
+//  Created by Noah Brauner on 8/15/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+import HealthKit
+
+struct GlucoseView: View {
+    @Environment(\.sizeCategory) var sizeCategory
+    var entry: SmallStatusEntry
+    
+    var body: some View {
+        VStack(alignment: .center, spacing: 0) {
+            HStack(spacing: 2) {
+                if let quantity = entry.currentGlucose?.quantity,
```

Comment:
This should be checking the age of the glucose, and indicating staleness. On the watch, we use three dashes ('---') .

Reply:
Stale glucose will now result it the glucose quadrant of the widget been grayed.